src/frontend/src/expr/agg_call.rs

```diff
@@ -98,6 +98,12 @@ impl AggCall {
             // Count
             (AggKind::Count, _) => DataType::Int64,
 
+            // StringAgg
+            (AggKind::StringAgg, _) => DataType::Varchar,
+
+            // SingleValue
+            (AggKind::SingleValue, [input]) => input.clone(),
```

Comment:
Reject multiple args here?

```suggestion
            (AggKind::SingleValue, [input]) => input.clone(),
            (AggKind::SingleValue, _) => return invalid(),
```